### PR TITLE
Media download

### PR DESCRIPTION
**Pull request by Matt95**
Adds button to download currently selected media as a zip.

Structure:
 - zip name = `{sceneID}_{sceneName}[_({searchTerm})].zip`  ([] = optional)
 - media are stored as `{guid}_{filename}`

### CODE REVIEW

File: src/js/utils/media-downloader.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+var JSZip = require("JSzip")
+
+zip = new JSZip();
```
Comment (aaronpcz) [inline src/js/utils/media-downloader.js]: The only thing I'd change in the commits so far.

I don't think you need this global variable (also could be abused if user spams download button).

You could make line 32, declare var zip and then just pass as a parameter to addUrlToZip.

Let me know what you think.

**Comment by Matt95** [comment]
Note: Download media will crash if the total size of the content is too large. 

For download audio themes the content is restricted to audio but if there are huge audio files the same issue will happen (unlikely)

File: src/js/utils/media-downloader.js (hunk 1 above)
Comment (Matt95) [reply src/js/utils/media-downloader.js]: Yep makes sense to make it non global

**Comment by aaronpcz** [comment]
Had a run through locally - all seems good with latest commits

**Pull request merged**